pom.xml

```diff
@@ -144,7 +144,7 @@
         <spring-security.version>5.1.5.RELEASE</spring-security.version>
         <tomcat.version>9.0.21</tomcat.version>
         <commons-io.version>2.4</commons-io.version>
-        <commons-collections.version>4.4</commons-collections.version>
+        <commons-collections.version>4.</commons-collections.version>
```

Comment:
did you remove this on purpose? the build will fail with "4." version

Reply:
Merge mistake I think